app-modules/contact/src/Models/Contact.php

```diff
@@ -235,6 +236,11 @@ public function organizations(): BelongsTo
         return $this->belongsTo(Organization::class, 'organization_id', 'id');
     }
 
+    public function user()
+    {
+        return $this->morphOne(KnowledgeBaseArticleVote::class, 'morphable');
```

Comment:
I don't think this relationship is quite right. A Contract can have a vote on multiple articles. So I believe this should be a Polymorphic many to many

Reply:
Done